dataland-frontend/src/components/forms/parts/fields/MultiSelectFormField.vue

```diff
@@ -7,7 +7,6 @@
       :validation-label="validationLabel ?? label"
       :placeholder="placeholder"
       :options="options"
-      inner-class="short"
```

Comment:
you need to pass "innerClass" to the `MultiSelectFormElement` component here I think.

Reply:
👍